docs/users-guide/05-visualizing-results.md

```diff
@@ -140,7 +140,8 @@ Another useful option for line, area, bar, and scatter charts is trend lines. If
 
 ![Trend lines](images/visualizations/trend-lines.png)
 
-#### Line Plus Bar charts
+#### Line + Bar charts <a name="line-plus-bar-charts">
```

Comment:
Is the lack of a closing `</a>` tag going to cause any problems?

Reply:
Probably! Fixed!